### ISSUE TITLE
Missing table header and incorrect heading level

### ISSUE THREAD

**Issue by ChiuchiuSorin** [xwikisas/application-mocca-calendar#136]
There are missing table headers at the `Subscribed calendars` livedata, as well as an inconsistency with the heading level of the title `Subscribed calendars` header.

**Comment by sarahxb** [comment]
I changed the header level from h6 to h3. The other accessibility issues mentioned in the ticket are related to the livedata and not to the mocca application
